Analytical_Engine/Query/IsContaining.cs

```diff
@@ -81,6 +83,40 @@ public static partial class Query
                 Item2 = notContained,
             };
         }
+
+        [Description("Defines whether an Environment Space contains each of a provided list of points.")]
+        [Input("region", "An Environment Space object defining a perimeter to build a 3D volume from and check if the volume contains the provided point.")]
+        [Input("regionHeight", "The height of the region.", typeof(BH.oM.Quantities.Attributes.Length))]
+        [Input("points", "The points being checked to see if it is contained within the bounds of the 3D volume.")]
+        [Input("acceptOnEdges", "Decide whether to allow the point to sit on the edge of the region, default false.")]
+        [Output("isContaining", "True if the point is contained within the region, false if it is not.")]
+        public static List<bool> IsContaining(this IRegion region, double regionHeight, List<Point> points, bool acceptOnEdges = false)
+        {
+            List<Polyline> panelsFromSpace = region.ExtrudeToVolume(regionHeight);
```

Comment:
This method could be much faster if you simply check whether the points fall within height domain and outline?

Reply:
Leaving out of this PR, can be picked up as a separate issue if necessary.